index.bs

```diff
@@ -174,25 +176,45 @@ Group members may propose a different process for their work item, which the cha
 
 ### Additional CCG Requirements
-The CCG has additional requirements requires the following in order to adopt a [=work item=]:
+The CCG has additional requirements in order to adopt a [=work item=]:
 - at least one group member willing to commit to being the first editor
 - another group member (from a different company) willing to be a co-editor
 - a suggested timeline
+- address the [=work item=] questions below in their [=proposal=]
+- a link to the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>
 - the expected [=work item=] [=type=]
 
 ### Work Item Scope
 In general, any work that advances the [[CCG-MISSION]] is welcome for consideration as a [=work item=]. 
 
+### Work Item Questions
+Each proposed [=work item=] must answer the following questions in their [=proposal=]. These questions are based on the Heilmeir Catechism. The intention is to improve clarity and understanding of the proposed [=work item=].
+- Explain what you are trying to do using no jargon or acronyms.
+- How is it done today, and what are the limits of the current practice?
+- What is new in your approach and why do you think it will be successful?
+- How are you involving participants from multiple skill sets and global locations in this work item? (Skill sets: technical, design, product, marketing, anthropological, and UX. Global locations: the Americas, APAC, Europe, Middle East.)
+- What actions are you taking to make this work item accessible to a non-technical audience?
+
+### Work Item Best Practices
+While not a requirement, the following best practices are recommended
+- peer leadership. Inspired from peer programming, peer leadership is teaming less experience work item owners with experienced work item owners for hands on experience. This enables knowledge transfer and empowers a larger number of future work item owners.
+
 ## Work Item / Target Deliverable Classification
 
 ### Categories
 
-CCG classifies its [=work item=]s by the [=target deliverable=]. The first relevant classification is a [=target deliverable=] [=category=]; the CCG distinguishes between two:
+CCG classifies its [=work item=]s by the [=target deliverable=]. The first relevant classification is a [=target deliverable=] [=category=]; the CCG distinguishes between three:
 - [=community report=]
 - [=ongoing community draft=]
+- [=task force=]
 
 The [=ongoing community draft=] category is not common in the W3C, but has emerged as a useful concept in the CCG to cover ongoing [=work item=]s that are useful to the community, while not being intended to turn into a [=final report=]
 
+The [=task force=] category is not common in the W3C, but has emerged as a useful activity in the CCG to discuss topics of importance to the community.  Task forces must
+- have at least one leader
+- provide a status report each month (meeting minutes will typically suffice)
```

Comment:
I vote to strike the status report as a "must" requirement for these reasons:
- Introduces possibly unnecessary process. 
- It's a burden on both the chairs to monitor this and the task force to provide status reports.
- Doesn't address what happens if the task forces fail to meet the "must" requirement

The charter and leader don't fall into this category because those only apply on creating, not ongoing requirements.

If we want ongoing requirements, which _may_ make sense, it should be lightweight but also describe what happens if the requirements are not met.

Reply:
@kimdhamilton those were community comments to the Task Force Guidelines doc that no one had any issues with when we discussed at Jan 6 meeting. Where does task force governance go if not in this document?

Task forces have to do minutes as part of being a task force, and the way it is written, the minutes can substitute for them. I had previously had more relaxed timeframe which was overwritten by you and others wanting a specific timeframe (multiple requests for monthly). Task forces must have accountability to the CCG through reporting - whether it is weekly, quarterly or from time to time. 

@wyc will you also weigh in here. Thanks.

Reply:
```
The [=task force=] category is not common in the W3C, but has emerged as a useful activity in the CCG to discuss topics of importance to the community. To create a task force, there must be
- at least one leader
- completed requirements for a [=work item=]

For a task force to remain active, it must
- provide a status update report within a timely manner of the chairs' request (meeting minutes will typically suffice)

If a response is not rendered within two weeks of the chairs' request, then the chairs may deem the task force to be inactive.
```

Reply:
Would this work for everyone?

Reply:
works for me

Reply:
I think that works. We can get feedback from the community on the response time interval, but I like the idea of spelling out that it's for determining whether the group is active (vs just imposing a process burden for chairs and the task force)